examples/wip/first-person-map/app.js

```diff
@@ -22,8 +23,10 @@ import {StaticMap} from 'react-map-gl';
 
 // Source data CSV
 const DATA_URL = {
-  BUILDINGS: 'https://raw.githubusercontent.com/uber-common/deck.gl-data/master/examples/trips/buildings.json',  // eslint-disable-line
-  TRIPS: 'https://raw.githubusercontent.com/uber-common/deck.gl-data/master/examples/trips/trips.json'  // eslint-disable-line
+  BUILDINGS:
+    'https://raw.githubusercontent.com/uber-common/deck.gl-data/master/examples/trips/buildings.json', // eslint-disable-line
```

Comment:
Why this change? Should we drop the `eslint-disable-line` since this was only intended to allow us to break the length limit? If we are reformatting to avoid breaking line max, then drop.